apps/prairielearn/src/lib/editors.ts

```diff
@@ -1132,6 +1134,12 @@ export class QuestionTransferEditor extends Editor {
     infoJson.title = questionTitle;
     infoJson.uuid = this.uuid;
 
+    // When transferring a question from an example/template course, drop the tags. They
+    // are likely undesirable in the template course.
+    if (this.course.example_course || this.course.template_course) {
+      delete infoJson.tags;
+    }
```

Comment:
This is for `QuestionTransferEditor`. We could always unconditionally drop tags? I'm not sure what's best here.

Reply:
I'm inclined to leave them for now. That is, I like your current code.